### PR TITLE
Bin: check illegal characters

### LINKED ISSUE #55: [BUG]paper init
Type:
- [x] Bug
- [ ] Feature
- [ ] Help   

Version:   
latest

Description:  
image

Folder names need to limit or Change the rules for creating folder.
when i use `paper init` and enter the folder name with blank spaces inside, it will create folders with the every input that separated by spaces.
and error with `no such file or directory`, the name with blank spaces inside will not be found.

### PR DESCRIPTION
**Pull request by unix**
## PR Checklist

- [x] Fix linting errors
- [x] Label has been added

## Change information
Fixed #55    
@DhyanaChina/zen  need review

### CODE REVIEW

File: src/bin/paper-init.ts [hunk 1]
```diff
@@ -27,6 +27,11 @@ const question = [{
 ;(async() => {
   const { folder } = await inquirer.prompt(promps)
   
+  if (/\s+|\\|\/|\^|\|/.test(folder)) {
+    console.log(`\n${chalk.yellow(folder)} contains illegal characters.`)
```
Comment (berubou) [inline src/bin/paper-init.ts]: contains unsupported spaces

Comment (berubou) [reply src/bin/paper-init.ts]: Is it necessary to explain what characters are illegal?

**Review by berubou — DISMISSED** [review]

File: src/bin/paper-init.ts (hunk 1 above)
Comment (unix) [reply src/bin/paper-init.ts]: agree

Comment (unix) [reply src/bin/paper-init.ts]: done

**Review by berubou — APPROVED** [review]

**Pull request merged**